SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+# Author: Rodrigo Sobral 2024
+# Copyright CERN
+# This script allows to create a venv virtual environment to use in notebooks and terminals. The environment contains the packages from a provided requirements file.
+
+# By default, we use the Python installed in the image
+ACCPY_BASE=$ACCPY_PATH/base
+ACCPY_ALL_VERSIONS_STR="?"
+PYTHON_PATH=$(which python)
+if [ -d "$ACCPY_PATH" ]; then
```

Comment:
Yeah, that's right, but this won't cause problems because some lines below this is done.

```bash
if [ -n "$ACCPY_CUSTOM_VERSION" ] && [ ! -e "$ACCPY_BASE/$ACCPY_CUSTOM_VERSION" ]; then
    >&2 _log "ERROR: Invalid Acc-Py version. Options: ${ACCPY_ALL_VERSIONS_STR}"
    exit 1
fi
```

Which means nothing will be done if the provided version doesn't exist.

Replying your question, I believe not. All the installers I used mount its version within a base folder.